10_bees_vs_wasps/10_bees_vs_wasps.jl

```diff
@@ -341,6 +341,22 @@ md"
 We see that the accuracy got a little better now, reaching a maximum of $85\%$. For now this will be enough, but there are many things we can do to improve the performance of the model. Hopefully this chapter had given some starting point for you to dig deeper into these topic if it is of your interest!
 "
 
+# ╔═╡ c838ce7e-8562-11eb-12a9-4d5fed79fb02
+md"
+### Summary
+In this chapter, we implemented a convolutional neural network that discriminated between pictures of bees and wasps.
+
+First, we saw how neural networks work and explained their training process, where we set a loss function that indicates how our model is performing and feed it with labeled data so that their parameters find the right setting to optimize the loss function. 
```

Comment:
replace: indicates
by: evaluates

Reply:
I think the clarity of this line can be improved. Maybe break it into two sentences, with a clear definition of loss function